### ISSUE TITLE
Auto login for guest

### ISSUE THREAD

**Issue by angelawong** [airsonic/airsonic#1159]
## Problem description
In Madsonic, I could create a link for my guests to allow them into the application without having them to login by doing the following:
http://madsonic1.domain.com/login.view?user=username&password=password

Can I do something simliar in Airsonic? Knowing how similar Airsonic is to Madsonic, I tried something similar in Airsonic but to no avail. Please advise. Thank you very much.

Airsonic 10.3.1
OS Debian 9
Proxy Apache2

**Comment by jvoisin** [comment]
Yes, it should be possible. Where would you like those links generated/displayed? Do you have some screenshots for Madsonic's interface?

**Comment by angelawong** [comment]
thearthur, just wondering why this request is closed? I was not asking that the admin password be changed. I want to be able to create a guest account (username=guest; password=guest-password) and allow people access to my albums without requiring them to log in. Basically creating a link like I can in Madsonic http://madsonic.domain.com/login.view?user=guest&password=guest-password. All they have to do is to click on the link and they are logged in automatically.

**Comment by eharris** [comment]
@angelawong This ticket was not closed.  A different ticket that (unintentionally) referenced this issue was closed.

**Comment by angelawong** [comment]
Thank you eharris. Am I right to assume that this feature will not appear until version 10.5.0? I know that we are now in version 10.4.0. When will version 10.5.0 come out?

**Comment by eharris** [comment]
Unless someone is willing to pick this up and submit a pull request, there's no telling when it might get worked on.

**Comment by neckroRO** [comment]
Hello. angelawong thanks for the request. I`m looking for something like this too. Admin, any ETA for implementation? Thank you!

**Comment by neckroRO** [comment]
http://MYIP:8080/airsonic/login.view/j_username=guest&j_password=guest won`t work neither.

**Comment by jvoisin** [comment]
As @eharris said:

> Unless someone is willing to pick this up and submit a pull request, there's no telling when it might get worked on.

**Comment by neckroRO** [comment]
Along with this, playlist option will need to be disabled. Because the access will become open, each anonymous login will be able to create one or more playlists.

**Comment by neckroRO** [comment]
jvoisin can you do it as contributor?

**Comment by freggal** [comment]
Using Madsonic with autologin in a Kiosk browser for guest users where there isn't any keyboard. This may be stale, but its still a relevant issue.
